cyclos-4-cluster/manifest.jps

```diff
@@ -80,7 +81,7 @@ actions:
     
     ConfigDB:
      - cmd[${nodes.sqldb[0].id}]: PGPASSWORD=${nodes.sqldb.password} psql -Uwebadmin postgres -c "create database cyclos4;"
-     - cmd[${nodes.sqldb[0].id}]: wget https://raw.githubusercontent.com/jelastic-jps/cyclos/master/cyclos-4-cluster/dumps/${settings.version}.bak -O cyclos4.bak 2>&1
+     - cmd[${nodes.sqldb[0].id}]: wget https://raw.githubusercontent.com/vlobzakov/cyclos/master/cyclos-4-cluster/dumps/${settings.version}.bak -O cyclos4.bak 2>&1
```

Comment:
Please use {baseUrl} placeholder instead of hardcoded link. In your case it could be merged on prod with **vlobzakov** repo

Reply:
{baseUrl} placeholder changed and database Url backup replaced with {baseUrl} placeholder.